util/gitutil/gitutil.go

```diff
@@ -196,3 +197,10 @@ func stripCredentials(s string) string {
 	ru.User = nil
 	return ru.String()
 }
+
+func ToAbs(path string) string {
+	if !filepath.IsAbs(path) {
+		path, _ = filepath.Abs(filepath.Join(osutil.GetWd(), path))
+	}
+	return SanitizePath(path)
+}
```

Comment:
Could be moved to `osutil` as well but needs to move `SanitizePath` there too, can be for a follow-up.